cypress/integration/views/credentials.view.ts

```diff
@@ -15,8 +15,8 @@ limitations under the License.
 */
 export const credentialNameInput = "input[name=name]";
 export const descriptionInput = "input[name='description']";
-export const usernameInput = "input[name=user]";
-export const passwordInput = "input[name=password]";
+export const usernameInput = "#user";
+export const passwordInput = "#password";
```

Comment:
@midays can you check if this locator was giving any error. 
This locator was changed in 
https://github.com/konveyor/tackle-ui-tests/pull/300/files#diff-36704bce7f817560ef7ae94310b30565873b6970140416f1d1e678aa0b272108R33

Reply:
This is difference between downstream and upstream, upstream is using ID so I changed to it. It will NOT work on downstream until it will be updated